backends/apple/metal/runtime/shims/et_metal_ops.mm

```diff
@@ -224,12 +224,15 @@ AOTITorchError aoti_torch_mps_mm_out(
       MPSDataType mps_dtype;
       size_t element_size;
 
-      ET_LOG(Debug, "aoti_torch_mps_mm_out: self_tensor scalar_type=%d, SupportedDTypes::FLOAT32=%d, SupportedDTypes::BFLOAT16=%d",
-             dtype, static_cast<int32_t>(SupportedDTypes::FLOAT32), static_cast<int32_t>(SupportedDTypes::BFLOAT16));
+      ET_LOG(Debug, "aoti_torch_mps_mm_out: self_tensor scalar_type=%d, SupportedDTypes::FLOAT32=%d, SupportedDTypes::FLOAT16=%d, SupportedDTypes::BFLOAT16=%d",
+             dtype, static_cast<int32_t>(SupportedDTypes::FLOAT32), static_cast<int32_t>(SupportedDTypes::FLOAT16), static_cast<int32_t>(SupportedDTypes::BFLOAT16));
 
       if (dtype == static_cast<int32_t>(SupportedDTypes::FLOAT32)) {
```

Comment:
I'm not very well familiar with ET coding practice, but considering you have no modify it in two places, why not have `to_mps_dtype(SupportedDtypes)` inline function and call it here and few hundrend lines down below?